runtime/permissions.rs

```diff
@@ -1976,10 +2012,10 @@ fn permission_prompt(message: &str) -> bool {
   // buffered data cannot effect the prompt.
   clear_stdin();
 
-  let opts = "[y/n (y = yes allow, n = no deny)] ";
+  let opts = "[yes / no] ";
```

Comment:
why make the options less informative of what can be passed?

Reply:
It's rather opaque with all the syntax: `/` `(` `=` `,`, `[`, `]`
I think this is easier to read

Reply:
then maybe `[(y)es / (n)o]`? i have seen this in a few clis

Reply:
I'll undo this change, it's unrelated.